server/cli/src/refresh_dates.rs

```diff
@@ -54,6 +55,9 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("sync_log", "pull_remote_finished_datetime"),
         ("sync_log", "integration_started_datetime"),
         ("sync_log", "integration_finished_datetime"),
+        ("temperature_breach", "start_timestamp"),
+        ("temperature_breach", "end_timestamp"),
+        ("temperature_log", "timestamp"),
```

Comment:
We usually use `datetime` instead of `timestamp`. Could they be renamed please? or is there something more to it?

Reply:
[2e009a4](https://github.com/openmsupply/open-msupply/pull/2326/commits/2e009a4a7de7738fde8a610e673135391ff725b9)